.github/workflows/packaging.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     if: |
       github.event_name == 'push' ||
       github.event.pull_request.head.repo.full_name != github.repository
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
I would leave a comment that briefly describes the problem and links the issue. Otherwise it is easy to conclude that this choice is arbitrary and step into the problem again.